### DISCUSSION TITLE
Slides: SBOMs - A short introduction

### DISCUSSION THREAD

**Discussion by mxmehl** [todogroup/ospology#376] [OSPO Stories]
Here you can find the slides for my presentation at the OSPOlogy Live event in October 2023. The topic was "SBOMs - A short introduction": [SBOMs - A Short Introduction - OSPOlogy.pdf](https://github.com/todogroup/ospology/files/12890745/SBOMs.-.A.Short.Introduction.-.OSPOlogy.pdf)

Abstract of the talk:

_Who hasn't heard of SBOMs? They seem to be ubiquitous and shiny tools for them are sprouting up everywhere. But what are they actually all about? What are the use cases? And what often causes practical application to fail? A short introduction for a common understanding, without marketing-speak._

Thanks for the great meet-up!

**Comment by anajsana** [comment]
Thank you!
